api/openapi-spec/conventions-server..yaml

```diff
@@ -0,0 +1,90 @@
+openapi: 3.0.0
+info:
+  title: Conventions server 
+  description: >-
+    "a sample conventions server"
+  version: 1.0.0
+  license:
+    name: Apache-2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+paths:
+  /webhook:
+    post:
+      description: "a sample conventions server"
+      responses:
+        200:
+          description: Successfully applied defined conventions
+          content:
+            "application/json":
+              schema:
+                $ref: '#/components/schemas/PodConventionContext'
+        400:
+          description: Error decoding PodConventionContext from request body or request body is nil 
+        500:
+          description: Internal server error
+          content:
+            "application/json":
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
```

Comment:
I don't think the client distinguishes between different types of http errors. Either the webhook call succeeds or it doesn't. Better alignment with Kubernetes by following the model for [AdmissionReview  response](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#response) would be good, but is a separate issue.